tox.ini

```
@@ -1,8 +1,8 @@
 [flake8]
-max-line-length = 120
+max-line-length = 80
 
 [tox]
-envlis = py34,py35,py36,py37
+envlis = py37
```

```suggestion
envlis = py3
```
if you use `py37` `tox` will fail if that exact version is not installed. the value `py3` will use whatever version of python3 is present,

---

well, I'm making use of `f-strings`, as you probably noted.  those are only supported in py37+.  still think `py3` is a better choice?

---

I think so. Since tox doesn't install python I think it's more appropriate to configure the exact python version with the tool that will install it (CI or python docker image).

The tests should fail pretty loudly if the version of python running them isn't new enough, and by letting the python3 version "float" you'll be able to run the tests across minor versions without needing to change the tox config.